Update dashboard CD workflow docs (#3426)
threefoldtech/tfgrid-sdk-ts: docs/workflows.md

### [CD](/.github/workflows/playground_cd.yml)

> On **Push** to development branch: It will do a clean install of node dependencies, cache/restore them, build the source code and deploy to staging server by Copying the artifacts using ssh to `staging.dashboard.dev.grid.tf`.

Reviewer: ```suggestion
On **Push** to the development branch, this workflow will do the following:
- clean install of node dependencies
- cache/restore dependencies
- build source code 
- copy artifacts to `staging.dashboard.dev.grid.tf` using SSH
- deploy to the staging server

```
